game.js

```diff
@@ -1065,8 +1065,18 @@ class NetworkHandler {
 }
 
 function resizeCanvas() {
-  canvas.height = canvas.clientHeight;
-  canvas.width = canvas.clientWidth;
+  const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
```

Comment:
we have isMobile at the top of minimalTest.js file and can use that

Reply:
(though it initializes just once so if you enter/leave mobile simulator mode in chrome you need to refresh the page)